doc-content/enterprise-only/processes/process-instance-migration-installing-service-proc.adoc

```diff
@@ -5,66 +5,36 @@ You can use the process instance migration (PIM) service to create, export and e
 
 .Prerequisites
 * You have defined processes in a backup-ed {PRODUCT} development environment.
+* Java Runtime Environment (JRE) version 11 or later is installed.
 
 .Procedure
 . Download the `{PRODUCT_FILE}-add-ons.zip` file from the {PRODUCT_DOWNLOAD_LINK}[Software Downloads] page for {PRODUCT} {PRODUCT_VERSION}.
 . Unzip the downloaded archive.
 . Move the `{PRODUCT_FILE}-process-migration-service-standalone.jar` file from the add-ons archive to a desired location.
-. In the location, create a YAML file containing the kieserver and Thorntail configuration, for example:
-+
-----
-thorntail:
-  deployment:
-    process-migration.war:
-      jaxrs:
-        application-path: /rest
-      web:
-        login-config:
-          auth-method: BASIC
-          security-domain: pim
-        security-constraints:
-          - url-pattern: /*
-            roles: [ admin ]
-          - url-pattern: /health/*
-  datasources:
-    data-sources:
-      pimDS:
-        driver-name: h2
-        connection-url: jdbc:h2:mem:test;DB_CLOSE_DELAY=-1;DB_CLOSE_ON_EXIT=FALSE
-        user-name: DS_USERNAME
-        password: DS_PASSWORD
-  security:
-    security-domains:
-      pim:
-        classic-authentication:
-          login-modules:
-            UsersRoles:
-              code: UsersRoles
-              flag: required
-              module-options:
-                usersProperties: application-users.properties
-                rolesProperties: application-roles.properties
-kieservers:
-  - host: http://localhost:8080/kie-server/services/rest/server
-    username: KIESERVER_USERNAME
-    password: KIESERVER_PASSWORD
-  - host: http://localhost:8280/kie-server/services/rest/server
-    username: KIESERVER_USERNAME
-    password: KIESERVER_PASSWORD1
-----
 . Start the PIM service:
 +
 [source,subs="attributes+"]
 ----
 $ java -jar {PRODUCT_FILE}-process-migration-service-standalone.jar -s./config.yml
 ----
-. To enable auto-detection of a JDBC driver by Thorntail, add the `JAR` file of the JDBC driver name to the `thorntail.classpath` system property. For example:
+. Ensure the relevant authentication method is set for `auth-method`. Valid values are `file`, `jdbc`, and `ldap`. For example:
 +
-[source,subs="attributes+"]
 ----
-$ java -Dthorntail.classpath=./h2-1.4.200.jar -jar {PRODUCT_FILE}-process-migration-service-standalone.jar -s ./config.yml
+database:
+  generation: update
+pim:
+  auth-method: file
 ----
+. To add dependencies or modify the build time properties by re-augmenting your Quarkus application, build your application with the `quarkus.package.type=mutable-jar` option:
```

Comment:
this is already set in the base configuration

Reply:
Removed